journal/modernized/02-chapter-1.adoc

```diff
@@ -487,10 +517,15 @@ and the approbation of President Washington having been obtained,
 they were deputed in its behalf to attend the said treaty,
 and present to the natives the following address:
 
-To the.
+[.embedded-content-document.address]
```

Comment:
So it seemed to me like this was an address that was sent, but there are other addresses in the following chapter that he seems to just be quoting. This seemed slightly different than those in how it it's a presented, but I could be misunderstanding it too.